example/build_cmake_installed/cmake_example.cpp

```diff
@@ -14,12 +14,14 @@
 //
 //@HEADER
 
+#include <Kokkos_Macros.hpp>
 #include <Kokkos_Core.hpp>
-
 #include <cstdio>
 #include <iostream>
 
+#ifndef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULE
 extern "C" void print_fortran_();
+#endif
```

Comment:
I'm not suer what the problem here is.
```suggestion
#ifndef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULE // FIXME
extern "C" void print_fortran_();
#endif
```